src/js/components/landing/recentActivity/RecentActivityTable.jsx

```diff
@@ -87,7 +87,7 @@ export default class RecentActivityTable extends React.Component {
 	
```

Comment:
Just commenting here because I want to comment on the entire file. I can't check right now because my config file is all messed up but does all of this happen properly on the recent activity table as well? Including not reloading when the modal opens?